README.md

```diff
@@ -16,7 +16,7 @@ For changes in each release, please check the releases page: <https://github.com
 
 ## Screenshots
 
-- _All screenshots assume the [global filter](#filtering-checklist-items) `#task` which is not set by default (see also [installation](#installation))._
+- _All screenshots assume the [global filter](https://obsidian-tasks-group.github.io/obsidian-tasks/getting-started/global-filter/) `#task` which is not set by default (see also [installation](#installation))._

```

Comment:
It's good that this links to the published docs, thank you.

I notice on the same line that the installation link has the same problem, so I'm curious why the linter didn't point that out. I think it should probably be fixed too.

Are there any similar relative links on this page that would also be broken?

Reply:
A way to test it would be to go to this plugin in Obsidian's Browser Plugins, and test all the links that are shown, as it is the Readme that is shown inside Obsidian.

Reply:
I tested, and the installation link indeed doesn't work inside Obsidian, so I think it would be worth making it link to:

https://obsidian-tasks-group.github.io/obsidian-tasks/installation/

I tested all the other links when viewed in Obsidain, and they are fine - and it of course confirms that the fix you made here was needed.

Would you mind fixing the Installation link to https://obsidian-tasks-group.github.io/obsidian-tasks/installation/ whilst you are here please? If not, no problem, I can do a separate PR.